components/LeagueEntries/LeagueEntries.test.tsx

```diff
@@ -0,0 +1,64 @@
+import { LeagueEntries } from './LeagueEntries';
+import { render, screen } from '@testing-library/react';
+import React from 'react';
+
+describe('LeagueEntries', () => {
+  it('renders without a pick made', () => {
```

Comment:
I would call this `renders 'default' state without a pick made`

Reply:
Updated in latest commit